src/shared/components/post/post-listing.tsx

```diff
@@ -741,9 +741,12 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
         <Icon icon="message-square" classes="me-1" inline />
         {post_view.counts.comments}
         {this.unreadCount && (
-          <span className="text-muted fst-italic">
-            ({this.unreadCount} {I18NextService.i18n.t("new")})
-          </span>
+          <>
+            {" "}
+            <span className="text-muted fst-italic">
```

Comment:
Can we safely remove the `text-muted`, since it's inside a `text-muted`?

Reply:
@jsit By looking at the code, yes, we can safely remove.